src/parser/jobs/rdm/Gauge.js

```diff
@@ -80,6 +92,17 @@ export default class Gauge extends Module {
 			black: [],
 		}
 
+		//Finisher Handling
+		_incorrectFinishers = {
+			verholy: 0,
+			verflare: 0,
+			bothprocsup: 0,
+		}
+
+		// The report timestamp is relative to the report timestamp, and in ms. Convert.
+		_parseDate = getPatch(this.parser.parseDate)
```

Comment:
Apparently this doesn't work in es6, blame Khloe :blobmorning:

Suggested putting it in the constructor or a getter.

Reply:
Did someone else do this in a working fashion already?  If so I'll just copy what they did.